### ISSUE TITLE
Several Questions about the paper content.

### ISSUE THREAD

**Issue by Dwrety** [LeonHLJ/Teach-DETR#1]
Hello, I have several questions about the ablations study section.
1. what does it mean by online teacher vs. offline teacher? Does online teacher means you also train the teacher as well? How do you train the teacher, by tracking the EMA? If so, this is self-distillation?
2. For the soft label vs. hard label section, have you confirmed it's because of the softmax of the MaskRCNN that causes the drop in performance. any experiments to verify it? 

Can the increase in performance be seen as a result of the increase in the Recall of the dataset's annotation, although that new annotation is noisy? Have you tried to train purely with the Aux boxes without GT. For instance, start with a H-DETR, train it regularly, and then train another H-DETR student using the outputs from the 1st model.

**Comment by LeonHLJ** [comment]
> Hello, I have several questions about the ablations study section.
> 
> 1. what does it mean by online teacher vs. offline teacher? Does online teacher means you also train the teacher as well? How do you train the teacher, by tracking the EMA? If so, this is self-distillation?
> 2. For the soft label vs. hard label section, have you confirmed it's because of the softmax of the MaskRCNN that causes the drop in performance. any experiments to verify it?
> 
> Can the increase in performance be seen as a result of the increase in the Recall of the dataset's annotation, although that new annotation is noisy? Have you tried to train purely with the Aux boxes without GT. For instance, start with a H-DETR, train it regularly, and then train another H-DETR student using the outputs from the 1st model.

1. Our approach does not involve training the teacher directly. In the case of an online teacher, we leverage the teacher forward process during training to obtain auxiliary boxes that are aligned with the augmented input image. On the other hand, in an offline teacher scenario, we obtain the auxiliary boxes beforehand and augment them alongside the ground truth (GT) boxes. 

In Table 8(b), we also explore using the Exponential Moving Average (EMA) model of the student as the teacher detector, which can be seen as a self-distillation process and further enhances our proposed strategy.

2. One possible explanation for the performance discrepancy between softmax and sigmoid detectors is the presence of an additional background class in the former. This can make it challenging to align the predictions of the two types of detectors accurately. While we have attempted several post-processing techniques to minimize this issue, our paper does not present the related results since they did not yield satisfactory results.

3. Do you mean the "increase in the Recall " is the newly-annotated objects in the dataset?  As shown in Sec. 3.3, the newly-annotated objects actually play a moderate role in boosting performance. Instead, the gain is multifaceted. The knowledge contained in the auxiliary boxes, such as the predicted classes, confidence scores, and box distributions, also plays a significant role in boosting performance. 

We appreciate your valuable suggestion, and while we have not experimented with training students solely on auxiliary boxes without GT, we hypothesize that it may lead to inferior results compared to using both sources of information.

**Comment by Dwrety** [comment]
I see, thanks for your clarification and quick reply. Interesting work.

**Comment by LeonHLJ** [comment]
Please do not hesitate to contact us if you have any questions or concerns.
